### DISCUSSION TITLE
Integrate snowswap with coingeco

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#77] [Integration]
Refer:

https://www.coingecko.com/en/coins/darwinia-network

**Comment by hackfisher** [comment]
Changed subswap to snowswap:

https://snowswap.xyz/#/

**Comment by nblogist** [comment]
https://github.com/orgs/dcdao/discussions/78#discussioncomment-8542743
adding an exchange will require the same process

CC @hackfisher

**Reply by hackfisher** [reply]
cc @1022ranji Need proof of affiliation with official project.

**Reply by 1022ranji** [reply]
I'll take care of this next week. @nblogist
